### PR TITLE
Group softdelete

### PR DESCRIPTION
**Pull request by alavieille**
[OO-FEATURE] Group is sotf deleteable
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/333
https://github.com/open-orchestra/open-orchestra/compare/add_functional_test?expand=1
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2039

### CODE REVIEW

File: Backoffice/Context/ContextManager.php [hunk 1]
```diff
@@ -115,7 +115,7 @@ public function getAvailableSites()
                 foreach ($user->getGroups() as $group) {
                     /** @var SiteInterface $site */
                     $site = $group->getSite();
-                    if (null !== $site && !$site->isDeleted()) {
+                    if (null !== $site && !$group->isDeleted() && !$site->isDeleted()) {
```
Comment (ngilain) [inline Backoffice/Context/ContextManager.php]: @alavieille : with this PR, what is the case where a site is deleted but not a group ?

File: UserAdminBundle/Controller/Admin/UserController.php [hunk 2]
```diff
@@ -165,7 +165,7 @@ protected function refreshLanguagesByAliases(UserInterface $user)
             foreach ($user->getGroups() as $group) {
                 /** @var SiteInterface $site */
                 $site = $group->getSite();
-                if (!$site->isDeleted() && !in_array($site->getSiteId(), $siteIds)) {
+                if (!$group->isDeleted() && !$site->isDeleted() && !in_array($site->getSiteId(), $siteIds)) {
```
Comment (ngilain) [inline UserAdminBundle/Controller/Admin/UserController.php]: @alavieille : same question here

File: Backoffice/Context/ContextManager.php (hunk 1 above)
Comment (alavieille) [inline Backoffice/Context/ContextManager.php]: normally never but it's a supplementary test

Comment (ngilain) [inline Backoffice/Context/ContextManager.php]: hum ok

**Comment by ngilain** [comment]
:+1:

**Pull request merged**